src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -94,6 +96,7 @@ public async Task SendAsync_SimpleGet_Success(Configuration.Http.RemoteServer re
 
         [OuterLoop("Uses external server")]
```

Comment:
Should we change these to `[OuterLoop("Uses external servers")]` as well, like in `src/libraries/Common/tests/System/Net/Http/PostScenarioTest.cs`

Reply:
unified everywhere